### ISSUE TITLE
Support disabling a whole class of rules

### ISSUE THREAD

**Issue by omnibs** [rubocop-hq/rubocop#8777]
## Is your feature request related to a problem? Please describe.

I'm frustrated because Chef callback scripts and Capistrano scripts in my codebase keep getting picked up by `rubocop-rails` rules, and when folks follow them, trying to be helpful, it causes problems. The most recent example is following `Rails/Pluck` breaking a Chef script.

## Describe the solution you'd like

I'd like to be able to say:

```
Rails/*:
  Exclude:
    - "deploy/**/*"
```

So Rails-related rules don't ever get called for non-Rails stuff.

## Describe alternatives you've considered

What we've been doing is adding `rubocop:disable` comments where the Rails rules hit, but when a new one comes along, some good-intentioned person follows it and breaks stuff.

We could add every `Rails/` rule to `.rubocop.yaml` with the exclude above, but it will also fail to capture the case where a new rule comes along and a good-intentioned person follows it and breaks stuff.

**Comment by koic** [comment]
RuboCop can set `Exclude` for each department. Is it possible to resolve this issue with the following setting for .rubocop.yml?

```yaml
Rails:
  Exclude:
    - "deploy/**/*"
```

**Comment by vlad-pisanov** [comment]
I believe you can achieve that by overriding the rules per directory with a custom `.rubocop.yml` file:

e.g.
`deploy/.rubocop.yml`
```yaml
inherit_from: ../.rubocop.yml
Rails:
  Enabled: false
```

**Comment by michaelglass** [comment]
@vlad-pisanov that works! thanks
